pyproject.toml

```diff
@@ -35,7 +35,7 @@ license = { text = "BSD-3-Clause" }
 requires-python = ">=3.6"
```

Comment:
Note for a separate discussion, I think this is still true, but we may want to consider moving to 3.8+ (I don't recall if we had some downstream users who were still on 3.6 though).

Reply:
We still have some users running Python 3.6, but they're also using an older UCX-Py version. I checked and they're ok with us moving to Python 3.8+ on current development version. This is now being addressed in https://github.com/rapidsai/ucx-py/pull/933 .